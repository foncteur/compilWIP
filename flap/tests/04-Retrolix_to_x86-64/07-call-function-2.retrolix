external printf

def print_two_int ()
  rax: %rax <- copy %rdi;
  rbx: %rbx <- copy %rsi;
  l10: %rdi <- copy "%d %d\n";
  l11: %rsi <- copy %rax;
  l12: %rdx <- copy %rbx;
  l13: %rax <- copy 0;
  l14: printf();
  l15: ret;
end

globals (x)
  l01: %rdi <- copy 37;
  l02: %rsi <- copy 73;
  l03: print_two_int();
  l04: exit;
end