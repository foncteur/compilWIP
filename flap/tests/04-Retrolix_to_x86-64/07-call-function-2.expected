Process exited with status exited(0).
STDOUT:
37 73

STDERR:



